### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by katfogy**
**Learning objectives**
-     Use React components.
-     Use React props.
-     Use React state.

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Amos, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has incorrect gitflow/ GitHub flow. Kindly note that the Pull Request for this activity should flow **from feature into development**. Kindly use the correct gitflow and request another code review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      <button type="button">3</button>
+      <button type="button" className="sign">+</button>
+      <button type="button" className="span-2">0</button>
+      <button type="button">.</button>
+      <button type="button" className="sign">=</button>
+    </div>
+  </div>
+);
+
+export default Calculator;
```
Comment (Gambit142) [inline src/components/Calculator.js]: - [ ] One of the project requirements is to create at least 1 child component of the `Calculator` component. Kindly create a child component and pass props from the `Calculator` component to the child component. 👍 💻

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @katfogy 👋, 

Good job so far!

## Highlights
- All linters checks passed ✔️ 
- Created `Calculator` component ✔️ 
- Created a `components` directory ✔️ 

You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️

- [ ] Kindly remove all boilerplates created by the **_create react app_** command. To be specific, please remove the following files:

- In the `public` directory:
  - logo192.png
  - logo512.png
  - manifest.json
  - robots.txt
- In the `root` directory:
  - App.css (because you created a Calculator.css file in the components directory which contains all styles needed)
  - App.test.js
  - index.css
  - logo.svg
  - reportWebVitals.js
  - setupTests.js

- [ ] _Kindly check the comments under the review for the other required change._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

You can also consider the following:

N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Gambit142 — APPROVED** [review]
Hi @katfogy 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Use of props ✔️ 
- Application's design is identical to that of the wireframe ✔️ 
- Use of props ✔️ 

### Optional Suggestion

N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**